### DISCUSSION TITLE
This is brilliant!

### DISCUSSION THREAD

**Discussion by dit7ya** [Artawower/orgnote#11] [General]
I am so pleased to stumble upon this project. My heart feels with joy and excitement to finally see someone bring org-roam (and not just org-mode) to not only to the web, but to Android as well!

I had the exact same frustration of not being able to write/ read to my org-roam database outside of Emacs, and thought of building basically this multiple times, but I had to give up every time eventually as there were just too many things to implement. 

I am gonna set up the Android app, try to sync my org folder, and write about my first impression soon.

Thank you so much for the project. Let me know if I can help with anything and wish you all the best!

**Comment by Artawower** [comment]
Thanks for the kind words, I really appreciate it. At the moment there is still a lot of work to be done, for the most part this is still a prototype, I want to collect all possible problems that can be encountered before I start to redesign the app.

At the moment synchronization does not work with the file system of android and ios devices, at the moment I am working on functionality that will allow synchronization of files regardless of their source, whether it is a browser or any other device that provides an interface to work with the file system.

The only option to synchronize notes is to use orgnote backend, it could potentially be self hosted (https://github.com/Artawower/orgnote-backend), or you can submit a request to waitlist here: https://about.org-note.com/ (please specify your nickname on github, as there are a lot of people interested now. Access is granted to those who are subscribed to patreon or have an active github account with contributions to any open source projects). The email may also get spammed.
